src/text.c

```diff
@@ -298,9 +299,10 @@ LedStateMaskText(struct xkb_context *ctx, enum xkb_state_component mask)
 
         mask &= ~(1u << i);
 
+        assert(LookupValue(lookup, 1u << i) != NULL);
```

Comment:
Maybe store the result in a var to avoid doing the lookup twice?

Reply:
Me too tired. Fixed